docs/resources/repository_binding.md

```diff
@@ -7,38 +7,65 @@ Manages [cyral repository to sidecar bindings](https://cyral.com/docs/sidecars/s
 ## Example Usage
 
 ```terraform
-resource "cyral_repository" "repo" {
-  name = "tf-account-repo"
-  type          = "mongodb"
+resource "cyral_sidecar" "sidecar" {
+  name = "my-sidecar"
+  deployment_method = "terraform"
+}
+
+resource "cyral_repository" "repo_mongodb" {
+  name = "my-mongodb-repo"
+  type = "mongodb"
   repo_node {
-        name = "single-node-mongo"
-        host = "mongodb.cyral.com"
-        port = 27017
+    name = "single-node-mongo"
+    host = "mongodb.cyral.com"
+    port = 27017
+  }
+  mongodb_settings {
+    server_type = "standalone"
   }
 }
 
-resource "cyral_sidecar" "sidecar" {
-  name = "tf-account-sidecar"
-  deployment_method = "docker"
+resource "cyral_repository" "repo_pg" {
+  name = "my-pg-repo"
+  type = "postgresql"
+  repo_node {
+    host = "pg.cyral.com"
+    port = 5432
+  }
 }
 
-resource "cyral_sidecar_listener" "listener" {
+resource "cyral_sidecar_listener" "listener_mongodb" {
   sidecar_id = cyral_sidecar.sidecar.id
   repo_types = ["mongodb"]
   network_address {
-    port          = 27017
+    port = 27017
   }
 }
 
-resource "cyral_repository_binding" "binding" {
+resource "cyral_sidecar_listener" "listener_pg" {
   sidecar_id = cyral_sidecar.sidecar.id
-  repository_id = cyral_repository.repo.id
-  enabled = true
+  repo_types = ["postgresql"]
+  network_address {
+    port = 5434 // clients will connect to the pg using sidecar port 5434
+  }
+}
+
+resource "cyral_repository_binding" "binding_mongodb" {
+  sidecar_id = cyral_sidecar.sidecar.id
+  repository_id = cyral_repository.repo_mongodb.id
   listener_binding {
-    listener_id = cyral_sidecar_listener.listener.listener_id
+    listener_id = cyral_sidecar_listener.listener_mongodb.listener_id
     node_index = 0
```

Comment:
Could add a note here that `node_index` is optional *if* there is only one repo node.